src/pages/upload/index.tsx

```diff
@@ -102,6 +129,25 @@ const Upload: React.FC = () => {
           </div>
         )
       })}
+      {ability.can('upload', 'Video.WithInstructor') ? (
+        <>
+          <h2>Insructor</h2>
+          {/* we can use a more featureful select component here that allows for search and displaying an image thumbnail. This is a proof of concept. */}
+          <select>
+            {instructors.map(
```

Comment:
Allow editor/publisher to select the instructor for this Course.

I'm dropping in a plain old `<select>` at the moment, but I think it'd be more useful to move to something more featureful that allows for search and displaying an avatar.